test/ui-test/testSuites/suite_wallet/tst_wallet/test.feature

```diff
@@ -48,16 +48,17 @@ Feature: Status Desktop Wallet
 
     @mayfail
     Scenario Outline: The user can manage a saved address
-        When the user adds a saved address named "<name>" and address "<address>"
-        And the user toggles favourite for the saved address with name "<name>"
-        Then the saved address "<name>" has favourite status "true"
-
-        When the user deletes the saved address with name "<name>"
-        Then the name "<name>" is not in the list of saved addresses
-
         When the user adds a saved address named "<name>" and address "<address>"
         And the user edits a saved address with name "<name>" to "<new_name>"
         Then the name "<new_name><name>" is in the list of saved addresses
+
+        When the user deletes the saved address with name "<new_name><name>"
+        Then the name "<new_name><name>" is not in the list of saved addresses
+
+        # Test for toggling favourite button is disabled until favourite functionality is enabled
```

Comment:
@IvanBelyakoff is it a task for enabling again the test?? If not, could you please create it?! Thankss!